### PR TITLE
cleanups in tests

### PR DESCRIPTION
**Pull request by eugene-matvejev**

### CODE REVIEW

File: src/pages/blog.test.jsx [hunk 1]
```diff
@@ -13,23 +13,19 @@ describe('<BlogIndex/>', () => {
...
+                                    aspectRatio: 1,
+                                    src: '{{src}}',
```
Comment (anasufana) [inline src/pages/blog.test.jsx]: Should we have a meeting about setting this as a standard when writing tests? It seems a bit unproductive to do this?

Comment (eugene-matvejev) [reply src/pages/blog.test.jsx]: to review snapshots with long URLs to make sure it inject what it suppose to inject, is hard :)

snapshot is only done to see what markup will be rendered and where probs will be injected

Comment (eugene-matvejev) [reply src/pages/blog.test.jsx]: @anasufana sure thing
👍

**Comment by andgreendigital** [comment]
I think a quick chat about this amongst the developers before this is approved!

File: src/pages/blog.test.jsx (hunk 1 above)
Comment (avi312singh) [reply src/pages/blog.test.jsx]: agree

**Pull request merged**